### ISSUE TITLE
Loss Function and training the model using YouTube-VOS.

### ISSUE THREAD

**Issue by NisTa24** [zlai0/MAST#22]
Hi @zlai0, thanks for the nice work and for sharing the code. I have a couple of questions and would appreciate your help.
1. What is the reason for multiplying the output and target with 20, before computing the loss (line 187 in main.py)? I could not find it anywhere in the paper and I wonder if this could interfere with the learning rate. 
2. For training the model on YouTube-VOS, I used  ytvos.csv. The training command is for the kinetics dataset (trains for 20 epochs as default) and I was wondering what should be changed in order to train the model from scratch on YoutubeVOS. Do I use only the frame pairs in ytvos.csv and pass the directory of the YouTube-VOS dataset as an argument?

Thank you in advance!
